lib/pg_query/node.rb

```diff
@@ -1,22 +1,27 @@
 module PgQuery
   # Patch the auto-generated generic node type with additional convenience functions
   class Node
+    def self.inner_class_to_name(klass)
+      @inner_class_to_name ||= descriptor.lookup_oneof('node').map { |f| [f.subtype.msgclass, f.name.to_sym] }.to_h
```

Comment:
https://bugs.ruby-lang.org/issues/15143
Can we use to_h with a block here? That was merged in 2018.

Reply:
Good point - looks like that was added in Ruby 2.6, so we could use it.